spacy/lang/fa/lemmatizer/_nouns_exc.py

```
@@ -3,779 +3,779 @@
+    "قبالات": ("قباله",),
+    "قرابات": ("قرابت",),
+    "مطلقات": ("مطلقه",),
+    "نزلات": ("نزله",),
+    "بکمان": ("بکیم",),
+    "روشان": ("روشن",),
+    "مسانید": ("مسند",),
+    "ناحیت": ("ناحیه",),
+    "رسوله": ("رسول",),
+    "دانشجویان": ("دانشجو",),
+    "روحانیون": ("روحانی",),
+    "قرون": ("قرن",),
```

Ruff (0.12.2)

Dictionary key literal `"قرون"` repeated

Remove repeated key literal `"قرون"`

(F601)